scripts/models/ngram_model/ngram_lm.py

```diff
@@ -0,0 +1,250 @@
+"""This module implements a ngram language model."""
+
+# import standard python modules
+from abc import ABC, abstractmethod
+import random
+from typing import Union, List, Tuple, Iterator
+import pickle
+import logging
+from pathlib import Path
+from itertools import tee
+from collections import defaultdict
+
+# import downloaded packages
+import numpy as np
+
+random.seed(1798)
```

Comment:
What is the purpose of setting the seed, and to that particular value? I could not find any reference to random in the files after this